Automattic/jetpack
# Fix JS error when installing boost plugin from recommendations

#### Changes proposed in this Pull Request:

* This PR fixes a JS error when installing the boost plugin from the recommendations interface.

#### Other information:


#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* You can reproduce the issue before checking out this branch
* Enable the boost conditional recommendation by following the test steps on #25431 
* Now, navigate to `/recommendations/summary`
* From the summary screen, click "Install" next to "Jetpack Boost" 
* Look in your browser console, you'll get an error that the method `finally` cannot be called on undefined.
* Deactivate the boost plugin from the plugins page.

* Now, checkout this branch
* Return to `/recommendations/summary` and click "Install" again for "Jetpack Boost"
* You should not get a JS error in the console anymore.
